src/clj/puppetlabs/services/config/puppet_server_config_service.clj

```diff
@@ -28,14 +29,17 @@
                               (get-in tk-config
                                       [:webserver :puppet-server]
                                       (get-in tk-config [:webserver]))
+                              (ca/initialize-ca-config (get-in tk-config
+                                      [:certificate-authority]))
                               puppet-config)
         (assoc context :puppet-config
                        {:puppetserver puppet-config}))))
 
   (get-config
     [this]
     (let [context        (tk-services/service-context this)
-          puppet-config  (:puppet-config context)]
+          puppet-config  (:puppet-config context)
+          tk-config      (get-config)]
```

Comment:
what's with this `tk-config`?

Reply:
good catch, I think that might be left over from some previous debugging I was doing.